2021/01/27 17:25:48 : categories_errors 2: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1113, in migration
    after_import = fn_after_import(id_desc, convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 1733, in after_category_import
    'thumbnail_id': thumbnail_id['data'][0]['ID'] if len(thumbnail_id['data']) > 0 else None
NameError: name 'thumbnail_id' is not defined

2021/01/27 17:25:48 : categories_errors 3: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1113, in migration
    after_import = fn_after_import(id_desc, convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 1733, in after_category_import
    'thumbnail_id': thumbnail_id['data'][0]['ID'] if len(thumbnail_id['data']) > 0 else None
NameError: name 'thumbnail_id' is not defined

2021/01/27 17:25:48 : categories_errors 4: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1113, in migration
    after_import = fn_after_import(id_desc, convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 1733, in after_category_import
    'thumbnail_id': thumbnail_id['data'][0]['ID'] if len(thumbnail_id['data']) > 0 else None
NameError: name 'thumbnail_id' is not defined

2021/01/27 17:25:48 : categories_errors 5: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1113, in migration
    after_import = fn_after_import(id_desc, convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 1733, in after_category_import
    'thumbnail_id': thumbnail_id['data'][0]['ID'] if len(thumbnail_id['data']) > 0 else None
NameError: name 'thumbnail_id' is not defined

2021/01/27 17:25:48 : categories_errors 6: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1113, in migration
    after_import = fn_after_import(id_desc, convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 1733, in after_category_import
    'thumbnail_id': thumbnail_id['data'][0]['ID'] if len(thumbnail_id['data']) > 0 else None
NameError: name 'thumbnail_id' is not defined

2021/01/27 17:25:48 : categories_errors 7: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1113, in migration
    after_import = fn_after_import(id_desc, convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 1733, in after_category_import
    'thumbnail_id': thumbnail_id['data'][0]['ID'] if len(thumbnail_id['data']) > 0 else None
NameError: name 'thumbnail_id' is not defined

